### DISCUSSION TITLE
Romulan revision and Romulan 2-page pdfs

### DISCUSSION THREAD

**Discussion by Felderburg** [bcholmes/StarTrek2d20#155] [Show and tell]
See below

[Romulan Char Sheet_v2_2 page.pdf](https://github.com/bcholmes/StarTrek2d20/files/11521665/Romulan.Char.Sheet_v2_2.page.pdf)
[Romulan Char Sheet_v9.4_page 1.pdf](https://github.com/bcholmes/StarTrek2d20/files/11521666/Romulan.Char.Sheet_v9.4_page.1.pdf)

**Comment by Felderburg** [comment]
I can create them as fillable, but I'm not sure what you need to make them into tool-ready pdfs.

Edit: https://github.com/bcholmes/StarTrek2d20/blob/sta-complete/WebTools/src/helpers/sheets.ts (see reply, editing this for my own use rather than creating a whole new reply).

**Comment by bcholmes** [comment]
Mostly it's just a naming standard (and ensuring that text fields are text fields and stress is a checkbox). 

e.g.: the name populates a field named "Name", the values go in to fields called "Value 1", "Value 2", "Value 3", etc. Weapons have a few fields: "Weapon 1 name", "Weapon 1 dice", "Weapon 1 qualities", and so forth.

The only other things to worry about are that most fields have the font size set to Auto, except for the Attributes/Disciplines which are set to a fixed value. The two pager's talent area needs a bit of special handling.

If you look at the sheet.ts, it contains all of the special names.

**Comment by Felderburg** [comment]
I started doing a fillable version of these revised sheets. I'll attach them in a reply here.

I am also considering doing sheets for various other factions/species, but we'll see how that goes. Specifically, in order of likely work being done:

Dominion (possibly a Jem'Hadar-only variant? I'll have to look at some member species to see if Dominion iconography is different for various species, as we mostly saw Jem'Hadar ship computers)
Bajoran
Cardassian
Andorian
Vulcan
Tellarite (maybe; I don't recall seeing much of their stuff)
Ferengi?
Borg/Liberated Borg?
Alternate Fed, although I think I've seen some on other sites (Disco, Kelvin timeline, time cops (TCARS) are all visually distinct enough from LCARS as I recall)

**Reply by bcholmes** [reply]
Looking forward to it. I've had folks ask about ST:ENT-themed and TOS movie-themed sheets, if that influences your thinking at all.

**Reply by Felderburg** [reply]
I think those are two specifically that I've already seen, alhough I may be thinking of the Enterprise crew pack.

**Comment by Felderburg** [comment]
[Romulan Char Sheet_v2_2 page_fillable.pdf](https://github.com/bcholmes/StarTrek2d20/files/11581068/Romulan.Char.Sheet_v2_2.page_fillable.pdf)
[Romulan Char Sheet_v2_2 page.pdf](https://github.com/bcholmes/StarTrek2d20/files/11581069/Romulan.Char.Sheet_v2_2.page.pdf)

I think I got the names right for the fillable form. You may have to make edits, but it should be reasonably ready. (The blank one also corrects a minor art hiccup.)

**Reply by bcholmes** [reply]
Both of these are the 2-pager? One with fields, and one without? Is that your intention?

**Reply by Felderburg** [reply]
[Romulan Char Sheet_v2_1 page_fillable.pdf](https://github.com/bcholmes/StarTrek2d20/files/11583213/Romulan.Char.Sheet_v2_1.page_fillable.pdf)
[Romulan Char Sheet_v2_1 page.pdf](https://github.com/bcholmes/StarTrek2d20/files/11583214/Romulan.Char.Sheet_v2_1.page.pdf)
Yes, that was the intention of that post, but I forgot that you might also need the 1-pager. I included the base pdf since I don't know what you do to create the small preview image of the sheet.

**Reply by bcholmes** [reply]
Okay. I started with the 1-pager. I made a handful of revisions. I think these are the changes I made:

1. I changed the height of the text fields to be the same. Often the height constrains the font size, so if they're at different heights, the font sizes end up being distractingly different. I tend to use 0.1944 inches as my default height (yes, that's an odd number, but the fields often snap to that value easily).
2. I changed font size of the numeric values (attributes, disciplines, weapon dice) to 9, and centred them in their text boxes. I find that so long as I have a size 9 font and a roomy box size, then I don't get the "I can't see the values in Adobe Acrobat on Windows" problem.
3. I changed the case of the last word in "Weapon 1 name", "Weapon 1 dice", etc. The code uses a lower case word, there.
4. There's a defect in the PDF library that I use to manipulate the character sheets: it manifests when I try to change the Stress values. The defect only happens if the check box has a non-standard style. Your sheet removed the outline and background colour from the checkbox (which I get: the big clunky box is kinda ugly), but that causes the defect to happen and the Stress boxes don't get updated properly. So I removed your Stress boxes and re-created them using the standard styling.

I'm letting you know, in case you want to do more of these in the future. The easier the PDFs are to "drop in", the faster they go to deployment.
